grid.cpp

```diff
@@ -503,9 +503,9 @@ void balanceLoad(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid, S
       //counter which is updated in acceleration, otherwise we just
       //use the number of blocks.
 //      if (P::propagateVlasovAcceleration) 
-      mpiGrid.set_cell_weight(cells[i], mpiGrid[cells[i]]->parameters[CellParams::LBWEIGHTCOUNTER]);
+//      mpiGrid.set_cell_weight(cells[i], mpiGrid[cells[i]]->parameters[CellParams::LBWEIGHTCOUNTER]);
 //      else
-//         mpiGrid.set_cell_weight(cells[i], mpiGrid[cells[i]]->get_number_of_all_velocity_blocks());
+      mpiGrid.set_cell_weight(cells[i], (Real) (1 + mpiGrid[cells[i]]->get_number_of_all_velocity_blocks()));
```

Comment:
Has this slipped into the PR by accident, or is this actually intended to skip LBWEIGHTCOUNTER alltogether when load balancing?

Reply:
(LBWEIGHTCOUNTER is set to exactly the num of velocity blocks just before. That +1 might have a small effect on distorting our load balancing, but I'm not to worried about it.)

Reply:
(I'm fixing this along with some other interesting merge conflicts)

Reply:
I don't think this is accident, I remember that the original setup was causing issues with the split cases. Unfortunately, the discussion appears to have disappeared from the rocket chat after 6 moths.

Reply:
Well, we don't want to merge that as-it-is into Vlasiator dev, as that would wreck havoc without load balance. I suggest we do `(Real)1 + LBWEIGHTCOUNTER` for now, and add a comment about FS-split functionality here. How about that?

Reply:
Sure. It should work without splitting, but there may be then some issues with some splits that may not function correctly.